ironfish/src/rpc/routes/chain/snapshotChain.test.ts

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { Assert } from '../../../assert'
+import { getBlockSize, writeBlock } from '../../../network/utils/block'
+import { useMinerBlockFixture } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+import { SnapshotChainStreamResponse } from './snapshotChain'
+
+describe('Route chain/snapshotChainStream', () => {
```

Comment:
This needs a number of tests.

You want to handle all the obvious failure cases and parameters. For example

- `start` and `end` could not be implemented at all and the tests would pass
- are forks handled?
- does `maxBlocksPerChunk` do anything?

Reply:
The schema has been adjusted, and a new test that explicitly tests `maxBlocksPerChunk` has been added.

Regarding forks, this route will only return blocks on the main chain by way of `Blockchain.getHeaderAtSequence()`; forks would be returned if we used the alternative `Blockchain.getHeadersAtSequence()`.